spec/lib/pact_broker/api/pact_broker_urls_spec.rb

```diff
@@ -153,6 +153,8 @@ module Api
         context "when the metadata is not valid base64" do
           let(:encoded_metadata) { "%" }
 
+          before { allow(Base64).to receive(:decode64).and_raise(ArgumentError) }
```

Comment:
As of `Base64.decode64` documentation
```
Returns the Base64-decoded version of str. This method complies with RFC
2045. Characters outside the base alphabet are ignored.
```
Im not so sure if `decode64` can raise exception or not hence the stubbing of
that method. Maybe a bit too implementation detail?

Reply:
Ah, yeah, I added the extra Base64.decode64 the other day without checking the whole spec. I'll update that expectation.